doc/debugging.rst

```diff
@@ -14,33 +14,108 @@ This allows breakpoints and the execution flow to be viewed and controlled right
 Debugging with ipdb
 ===================
 
-Spyder offers the following debugging features integrated into the native GUI:
+You can fully control debugger execution from the :guilabel:`Debug` menu, :guilabel:`Debug toolbar` and via configurable keyboard shortcuts, along with the standard ``ipdb`` `console commands`_.
 
-* Multiple means of setting and clearing normal and conditional breakpoints for any line in a file opened in the :doc:`editor`.
+.. image:: images/debugging/debugging-commands.gif
```

Comment:
This otherwise looks good, but after stepping the GIF loops back to the beginning almost instantly, giving the user no time to actually take in what just happened and leaving them confused that things jumped. Instead, I'd pause for probably a full two seconds to allow them to see and understand the effect of the next command before it jumps back.

Reply:
Agreed.

Reply:
This is now fixed, thanks.